R/fit.R

```diff
@@ -85,6 +97,15 @@ CmdStanFit <- R6::R6Class(
 
     # print summary table without using tibbles
     print = function(variables = NULL, ..., digits = 2, max_rows = 10) {
+      if (!length(self$output_files(include_failed = FALSE))) {
+        if (self$runset$method() == "sample") {
+          stop("No chains finished successfully. Unable to print.")
+        } else if (self$runset$method() == "generate_quantities") {
+          stop("Generating quantities for all MCMC chains failed. Unable to print.", call. = FALSE)
+        } else {
+          stop("Fitting failed. Unable to print.")
+        }
```

Comment:
Same question as above: what about simplifying this and using `stop("Fitting failed. Unable to print.")` for all methods?